### PR TITLE
Added the options -r, -n, -p, -s and updated Readme accordingly.

### PR DESCRIPTION
**Pull request by LoopingLudwig**
Hi,

thanks for your work on this.
I added some options for personal preference and for my use case of having reserved ASN-ranges for existing folders and printing them as needed.

- Option -r to increment from left to right, just a personal preference
- Option -n/-s to partly use label sheets (-n, -s)
- Option -p as it was requested [here](https://github.com/entropia/paperless-asn-qr-codes/issues/7) and I was on the way

### CODE REVIEW

**Review by margau — COMMENTED** [review]
Looks good to me at the first look, thanks!
Can you squash this again?
I'd also wait for a review from @jcgruenhage

**Review by jcgruenhage — APPROVED** [review]

**Review by margau — APPROVED** [review]

**Pull request merged**
